src/devices/src/legacy/rtc_pl031.rs

```diff
@@ -142,10 +142,12 @@ impl RTC {
                 self.trigger_interrupt()?;
             }
             RTCICR => {
-                // As per above mentioned doc, the interrupt is cleared by writing any data value to
-                // the Interrupt Clear Register.
-                self.ris = 0;
-                self.trigger_interrupt()?;
+                // The RTCICR is used together with the RTCMR register to implement a basic time alarm function.
+                // As per the above comment, we do not implement this functionality in firecracker. It can
+                // be easily checked that the value of self.ris will never change. Just like we do
```

Comment:
In this case, the value can actually change, if the driver does a write to RTCICR.

Reply:
`self.ris` is never set to 1 by our code, so any value that would be written (0 or 1) it cannot change to anything other than 0, right?
In a scenario where we would make use of the match functionality, self.ris would be asserted when the match value would equal the one desired.